index.html

```diff
@@ -32,5 +32,127 @@ <h1 class="playfair">Okolo <br> Oshoke</h1>
+                    <li class="li-jav poppins">Javascript</li>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+        <div class="grid-item-6">            
+            <div class="project-div">
+                <img src="img/project_5.jpg" alt="">
+                <h2 class="poppins">Project name goes here</h2>
+                <ul>
+                    <li class="li-htm poppins">HTML/CSS</li>
+                    <li class="li-rub poppins">Ruby on Rails</li>
+                    <li class="li-jav poppins">Javascript</li>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+        <div class="grid-item-7">            
+            <div class="project-div">
+                <img src="img/project_6.jpg" alt="">
+                <h2 class="poppins">Project name goes here</h2>
+                <ul>
+                    <li class="li-htm poppins">HTML/CSS</li>
+                    <li class="li-rub poppins">Ruby on Rails</li>
+                    <li class="li-jav poppins">Javascript</li>
+                </ul>
+                <button type="button"><span>See this project &#x2192;</span></button>
+            </div>
+        </div>
+    </section>
+    <!--About Section-->
+    <section class="mobile mobile-about">
+        <div>
+            <h1 class="playfair">About</h1>
+            <p class="poppins">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore 
+                magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+            </p>
+            <button type="button">
+                Get my CV
+                <span><img src="img/ic_download.svg" alt=""></span>
+            </button>
+        </div>
+        <div class="about-entry languages poppins">
+            <h3>Languages</h3>
+            <ul>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+            <img src="img/lang.svg" alt="">
+        </div>
+        <div class="about-entry frameworks poppins">
+            <h3>Frameworks</h3>
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby  on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+            <img src="img/frame.svg" alt="">
+        </div>
+        <div class="about-entry skills poppins">
+            <h3>Skills</h3>
+            <ul>
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>GitHub</li>
+                <li>GitLAb</li>
+                <li>Terminal</li>
+            </ul>
+            <img src="img/skills.svg" alt="">
+        </div>
```

Comment:
Please make sure the technology categories are wrapped in a `<ul>`, each of them in an `<li>`.

<img width="680" alt="Screen Shot 2022-06-29 at 18 05 51" src="https://user-images.githubusercontent.com/82426682/176560229-be266b41-71ae-4bb3-9d1d-ba7f11f5a274.png">